tinybmp/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 
 ### Added
 
+- Added support for decoding 1 bit pixel depth BMP images.
```

Comment:
Please add an `###Added` header to the `[Unreleased]` section and move this there. And you should also add the PR number in front of it:
```suggestion
- #352 Added support for decoding 1 bit pixel depth BMP images.
```